org.soundconverter.SoundConverter.yml

```diff
@@ -0,0 +1,47 @@
+app-id: org.soundconverter.SoundConverter
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+rename-appdata-file: soundconverter.appdata.xml
+rename-desktop-file: soundconverter.desktop
+rename-icon: soundconverter
+command: soundconverter
+finish-args:
+  # X11
+  - --socket=x11
+  # Wayland
+  - --socket=wayland
+  # Users should be able to edit files anywhere in their home
+  - --filesystem=home
```

Comment:
Unless it's something insists on, I'd prefer `xdg-music`, `xdg-download`, `/media` and `/run/media`.

Reply:
For now, I have changed it according to your suggestion, since the app does not require any specific location to work correctly.

Though, from my own perspective as user of flatpak apps, I often struggle with flatpaks providing some generic functionality, which then only allow you to open files stored in specific directories. I understand the idea of limiting access only to the bare minimum, but for generic apps like SoundConverter, this often disrupts user workflows: They have to move/copy files into some "allowed path", do the thing inside the application, and then move/copy them back to their original location again.

Personally, I would prefer `home` as it gives users a maximum of flexibility which of their sound files they are allowed to convert. I am often converting sound files which are not necessarily music, nor downloaded from the internet, nor coming from external media.